libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFSmartStore/SFSmartStorePlugin.m

```diff
@@ -514,4 +513,8 @@ - (NSString *)internalCursorId:(NSString *) cursorId withGlobal:(BOOL) isGlobal
 - (void)dealloc {
     SFRelease(_cursorCache);
 }
+- (void)pgGetSoupSpec:(nonnull CDVInvokedUrlCommand *)command {
```

Comment:
@wmathurin do you think we could get rid of this method in the public header if it hasn't had a definition?

Reply:
Yes the method can go. We removed it on the javascript side of the plugin already (see https://github.com/forcedotcom/SalesforceMobileSDK-Shared/pull/531/files).

Reply:
I should have removed it in https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Hybrid/pull/211/files

Reply:
Wait I did remove it: https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Hybrid/pull/211/files#diff-515f96aa6001c7aa00d80352293a737c3ef74dbc17fa8f48e1d7cef4e8ae1332L416
Someone must have merged it back.

Reply:
Wait it's coming back with this PR. We shouldn't have that method.

Reply:
Ah OK.  I can update this.